developer-concepts/readme.adoc

```diff
@@ -873,7 +877,7 @@ This job prints the current timestamp and the message "`Hello World`" every minu
 
 Create the Cron Job as shown in the command:
 
-	$ kubectl create -f template/cronjob.yaml --validate=false
+	$ kubectl create -f templates/cronjob.yaml --validate=false
```

Comment:
This should partially fix https://github.com/arun-gupta/kubernetes-aws-workshop/issues/113